README.md

```diff
@@ -1,70 +1,67 @@
-# Getting Started with Create React App
+# Crypto Metrics
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This is a mobile web app that  provides simple information about data in the crptocurrency market.
+User can see the current price of any available cryptocurrency, can see the market cap of any cryptocurrency coin and also filter the data based on the users's search.
+  
+![Crypto](./src/components/assets/crptocurrency.png)
 
-## Available Scripts
+## Live demo
+[CryptoMetrics](https://crypto-metrics-d.netlify.app)
```

Comment:
- [ ] Please, make sure that your deployment is working correctly. As of right now is giving a 404 error.

![image](https://user-images.githubusercontent.com/79114233/142741637-82809f2d-7365-4e06-a84f-8c245c8cac78.png)